js/everything.js

```diff
@@ -221,6 +229,13 @@ var ZenPlayer = {
                     }
                     window.sessionStorage[videoID] = resumeTime;
                 }
+                if (resumeTime > 0) {
+                    var updatedUrl = that.videoUrl + "&t=" + Math.round(resumeTime);
+                    $("#zen-video-title").attr("href", updatedUrl);
+                }
+                else if (resumeTime <= 0 && $("#zen-video-title").attr("href") !== that.videoUrl) {
+                    $("#zen-video-title").attr("href", that.videoUrl);
+                }
```

Comment:
Let's clean this up so we only set the `href` attribute once, using the if/else to change the `updateUrl` value then set it afterwards